XRTK-Core/Packages/com.xrtk.core/Inspectors/ControllerPopupWindow.cs

```diff
@@ -467,7 +467,7 @@ private void RenderInteractionList(SerializedProperty interactionList, bool useC
 
                 if (useCustomInteractionMapping)
                 {
-                    EditorGUILayout.LabelField($"{i + 1}", GUILayout.Width(32f));
+                    EditorGUILayout.LabelField($"{i}", GUILayout.Width(32f));
```

Comment:
What does this change exactly?

Reply:
Have a look at the screenshot below. Without this fix the leftmost column would display IDs starting at 1 ... 9 while the IDs of the mapping are actually 0 ... 8. You understand? If it's an ID column it should display the actual value of the ID.

![image](https://user-images.githubusercontent.com/9565734/78422919-1b4b9700-7663-11ea-8500-16bbf3dfd8ae.png)

Reply:
Seems fair, but can we break out any non-hands style fixes into their own PR?

Reply:
I'd like to fix this in #379